BoxContentSDK/BoxContentSDK/External/KeychainItemWrapper/BOXKeychainItemWrapper.m

```diff
@@ -250,10 +250,13 @@ - (NSMutableDictionary *)secItemFormatToDictionary:(NSDictionary *)dictionaryToC
     // Add the proper search key and class attribute.
     [returnDictionary setObject:(id)kCFBooleanTrue forKey:(id)kSecReturnData];
     [returnDictionary setObject:(id)kSecClassGenericPassword forKey:(id)kSecClass];
-    
+    [returnDictionary removeObjectForKey:kSecAttrModificationDate];
+    [returnDictionary removeObjectForKey:kSecAttrCreationDate];
```

Comment:
On OS X a keychain search query that includes a modification date or creation date seems to never return any result. These are included in the result from SecItemCopyMatching that's passed into this method, but shouldn't be necessary to find the correct item, so I've excluded them.